README.md

```diff
@@ -39,6 +39,7 @@ This Action is perfect for running on demand via `workflow_dispatch` or regularl
 * `REPO_FOR_DATA`: A repository to store data to. It will be stored in a `.data` directory.
 * `TRACK_ISSUES` (optional): `true` if you'd like to be notified about comments on issues
 * `LABELS` (optional): a list of labels that you'd like to track.
+* `DESCRIPTION` (optional): a markdown formatted description of the board being tracked.
```

Comment:
Could we just pull the project body? https://docs.github.com/en/rest/reference/projects#get-a-project I'm not sure there's a case where I'd want a seperate one in the workflow file as opposed to the project board... but perhaps there is.

Less inputs feels simpler if that works for you.

Reply:
👍  Works for me and that's probably a cleaner way of handling things. Would the project description be something this action should show by default or would it make more sense as opt in?

Reply:
By default is fine I think, let's see how it looks.